AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -124,63 +109,24 @@ + (BOOL)isSizingQueue {
   return kASSizingQueueContext == dispatch_get_specific(kASSizingQueueContext);
 }
 
-/**
- * Concurrent queue for query / updating the cached data.
- * The data query is more frequent than the data updating, so we use dispatch_sync for reading, and dispatch_barrier_async for writing.
- */
-+ (dispatch_queue_t)dataUpdatingQueue
-{
-  static dispatch_queue_t dataUpdatingQueue = NULL;
-  static dispatch_once_t onceToken;
-  dispatch_once(&onceToken, ^{
-    dataUpdatingQueue = dispatch_queue_create("com.facebook.AsyncDisplayKit.ASDataController.dataUpdatingQueue", DISPATCH_QUEUE_CONCURRENT);
-    dispatch_queue_set_specific(dataUpdatingQueue, kASDataUpdatingQueueContext, kASDataUpdatingQueueContext, NULL);
-  });
-
-  return dataUpdatingQueue;
-}
-
-+ (BOOL)isDataUpdatingQueue {
-  return kASDataUpdatingQueueContext == dispatch_get_specific(kASDataUpdatingQueueContext);
-}
-
 - (void)asyncUpdateDataWithBlock:(dispatch_block_t)block {
```

Comment:
since `-asyncUpdateDataWithBlock:` is now a non-blocking call, do we have to dispatch_async() to the sizing queue before invoking it?